packages/amplify_authenticator/lib/src/state/authenticator_state.dart

```diff
@@ -454,11 +454,8 @@ class AuthenticatorState extends ChangeNotifier {
   }
 
   /// Change to a new step in the authentication flow
-  void changeStep(AuthenticatorStep step, {bool resetAttributes = true}) {
+  void changeStep(AuthenticatorStep step) {
     _authBloc.add(AuthChangeScreen(step));
-
-    /// Clean [ViewModel] when user manually navigates widgets
-    if (resetAttributes) _resetAttributes();
```

Comment:
Is this change related to an issue in one of the tests?

Reply:
This was just clean up. It wasn't used anywhere.

Reply:
`resetAttributes` was defaulting to true. If that option is not used, shouldn't just the if statement be removed?

```dart
void changeStep(AuthenticatorStep step) {
  _authBloc.add(AuthChangeScreen(step));
}
```